research/conditional/utils/argparse.py

```diff
@@ -101,6 +101,9 @@ def introduce_parser_arguments(
     parser.add_argument("--git_branch", type=str, default=None)
     parser.add_argument("--decoding_interval", type=int, default=0)
 
+    parser.add_argument("--model_fits_filename", type=str, default=None)
+    parser.add_argument("--model_fits_params", type=str, default=None)
```

Comment:
rename it to gpu_usage_report_params
```suggestion
    parser.add_argument("--gpu_usage_report_params", type=str, default=None)
```

Reply:
remember to refactor